python/ray/_private/runtime_env/context.py

```diff
@@ -12,10 +12,12 @@ class RuntimeEnvContext:
 
     def __init__(self,
                  command_prefix: List[str] = None,
+                 container_command_prefix: List[str] = None,
```

Comment:
please don't change this. you can modify `self.py_exectuable` or `self.command_prefix` in ContainerManager.setup instead of adding a new entry.